recipes/metacache/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
   host:
     - gawk
     - wget
+    - zlib
```

Comment:
`zlib` needs to be a build dependency. Remind me later to do that.

Reply:
no, host is ok. It should not be in build and can be skipped in run.

Reply:
What explains the CI failure?

Reply:
You could try this: https://bioconda.github.io/contributor/troubleshooting.html#zlib-errors